src/current/cockroachcloud/network-authorization.md

```diff
@@ -1,45 +1,52 @@
 ---
-title: Network Authorization
+title: Network Authorization for CockroachDB Cloud clusters
 summary: Learn about the network authorization features for {{ site.data.products.db }} clusters.
 toc: true
 docs_area: manage
 ---
 
-{{ site.data.products.dedicated }} requires  explicit authorization of the networks that can access your cluster. This deny-by-default policy protects the cluster from network attacks including denial-of-service and password brute-forcing.
+{{ site.data.products.dedicated }} allows explicit configurations of the networks that can access your cluster.
 
-You can do this by:
+To prevent denial-of-service attacks, brute force password attacks, and other forms of malicious activity, Cockroach Labs recommends restricting your network to allow access only from specific IP address ranges controlled by your organization, according to the [Principle of Least Privilege (PoLP)](https://en.wikipedia.org/wiki/Principle_of_least_privilege). These might include specific networks for your application servers, hardened administrator access points, and backup-restore pipelines for disaster recovery.
 
-- [Adding an authorized range of public IP addresses](#ip-allowlisting).
-- Setting up [Google Cloud Platform (GCP) Virtual Private Cloud (VPC) peering](#vpc-peering) or [Amazon Web Service (AWS) PrivateLink](#aws-privatelink) for your cluster.
+## Options for controlling network access
 
-**Prerequisite**: {% include cockroachcloud/cluster-operator-prereq.md %}
+You can authorize network access to your cluster by:
 
-{{site.data.alerts.callout_success}}
-For additional cluster security, you can learn more about [Private Clusters (Preview)](private-clusters.html). A private cluster's nodes have no public IP addresses.
+- [Adding an authorized range of public IP addresses](#ip-allowlisting).
+- Setting up [Google Cloud Platform (GCP) Virtual Private Cloud (VPC) peering](#vpc-peering) or [Amazon Web Service (AWS) PrivateLink](#aws-privatelink) for your cluster (Dedicated clusters only). Access via [GCP VPC peering](#vpc-peering) or [AWS PrivateLink](#aws-privatelink) avoids traversing the public internet, and therefore offers several advantages:
```

Comment:
If this page is for serverless and dedicated, I just want it to be clear that VPC peering and PrivateLink are not supported for serverless. PrivateLink is coming soon, but VPC peering will never work with serverless.

Reply:
It does say "(Dedicated clusters only)". Not sure how to flag this harder, could use a callout element but that would kind of break up the list flow. It also flags that these features are ded only in the first sentence of the description section for each of the features. I feel it's sufficiently flagged, but perhaps @kathancox has a suggestion for making it clearer. thanks!